saleor/static/dashboard-next/discounts/views/VoucherCreate.tsx

```diff
@@ -65,7 +66,16 @@ export const VoucherDetails: React.StatelessComponent = () => {
                       endDate: formData.hasEndDate
                         ? joinDateTime(formData.endDate, formData.endTime)
                         : null,
-                      minAmountSpent: parseFloat(formData.minAmountSpent),
+                      minAmountSpent:
+                        formData.requirementsPicker === "NONE" ||
+                        formData.requirementsPicker === "ITEM"
+                          ? 0
+                          : parseFloat(formData.minAmountSpent),
+                      minCheckoutItemsQuantity:
+                        formData.requirementsPicker === "NONE" ||
+                        formData.requirementsPicker === "ORDER"
+                          ? 0
+                          : parseFloat(formData.minCheckoutItemsQuantity),
```

Comment:
It's the same as 
https://github.com/mirumee/saleor/pull/4556/files/9fd99d048ad050ea247a01fef15305aeea6375bb#diff-95dd24f9158deb91b0d37a41f830804cR339
so it can be extracted to helper function at the section level